### PR TITLE
fix(web): re-render of Reactivelist due to analytics prop

### PR DESCRIPTION
**Pull request by jyash97**
Dependent on: https://github.com/appbaseio/reactivecore/pull/60

### CODE REVIEW

File: packages/web/src/components/result/ReactiveList.js [hunk 1]
```diff
@@ -882,18 +863,16 @@ const mapStateToProps = (state, props) => ({
 			&& state.selectedValues[props.componentId].value - 1)
 		|| -1,
 	hits: state.hits[props.componentId] && state.hits[props.componentId].hits,
-	aggregationData: state.compositeAggregations[props.componentId] || [],
+	aggregationData: state.compositeAggregations[props.componentId],
```
Comment (bietkul) [inline packages/web/src/components/result/ReactiveList.js:866]: @jyash97 What's the need of this change?

File: packages/web/src/components/search/DataSearch.js [hunk 2]
```diff
@@ -1001,13 +979,11 @@ const mapStateToProps = (state, props) => ({
 			&& state.selectedValues[props.componentId].value)
 		|| null,
 	suggestions: state.hits[props.componentId] && state.hits[props.componentId].hits,
-	aggregationData: state.compositeAggregations[props.componentId] || [],
+	aggregationData: state.compositeAggregations[props.componentId],
```
Comment (bietkul) [inline packages/web/src/components/search/DataSearch.js:982]: @jyash97 Same here

**Review by bietkul — CHANGES_REQUESTED** [review]

File: packages/web/src/components/search/DataSearch.js (hunk 2 above)
Comment (jyash97) [reply packages/web/src/components/search/DataSearch.js:982]: This is causing an extra re-render because of the fallback.

Comment (bietkul) [reply packages/web/src/components/search/DataSearch.js:982]: @jyash97 I'm not sure if @ShahAnuj2610 handled it well.
@ShahAnuj2610 Can you confirm if this change wouldn't break the things?

**Pull request merged**